libcudacxx/test/libcudacxx/std/numerics/complex.number/complex.transcendentals/pow_complex_complex.pass.cpp

```diff
@@ -51,7 +51,10 @@ __host__ __device__ void test_edges()
       }
       else
       {
-        assert(real(r) == real(z));
+        if(real(r) != real(z))
+        {
+          is_about(real(r), real(z));
+        }
```

Comment:
We should either completely loosen the assertion or fix the code. I am for loosening it if it is not a required equality
```suggestion
        is_about(real(r), real(z));
```

Reply:
We're comparing `pow(i, j)` vs `exp(j * log(i))`. If the compiler does not optimize the comparison away, and we actually have to do exp, log and multiplication, there's no practical way not to lose precision.

If it helps, I can make a list of the values that we happen to trip on right now, and by how much the result happens to be different. The failure I poked at had the result difference of ~1e-18.

Reply:
Looks like the numeric differences only affect `double`, but they are pretty widespread:
https://gist.github.com/Artem-B/19807d5e4f2e3efb0e3d6319ebed1809

Reply:
For whatever reason I'm having trouble reproducing the issue on the host today, so it could be caused by my hacks to debug the issue. The original failure only reported the failure on a GPU. So, let's limit the scope to `double` on GPU.